dietpi/dietpi-software

```diff
@@ -9878,6 +9922,42 @@ _EOF_
 
 		fi
 
+		software_id=186 # IPFS Node
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			G_EXEC mkdir /mnt/dietpi_userdata/ipfs
```

Comment:
Use `mkdir -p` or `[[ -d '/mnt/dietpi_userdata/ipfs' ]] ||` to not have a failure when the directory exists already.